fe/fe-core/src/test/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessorOlapTest.java

```diff
@@ -919,6 +919,37 @@ public void testFilterPartitionByRefreshNumber() throws Exception {
         starRocksAssert.dropMaterializedView(mvName);
     }
 
+    @Test
+    public void testFilterPartitionByAdaptiveRefreshNumber() throws Exception {
```

Comment:
You may add more Integration tests, in sql-test.
eg: `test_mv_refresh_list_partitions_basic`/`test_mv_refresh_with_union`

Reply:
Fixed